ivadomed/main.py

```diff
@@ -463,6 +474,10 @@ def run_main():
     # Get context from configuration file
     path_config_file = args.config
     context = imed_config_manager.ConfigurationManager(path_config_file).get_config()
+    
+    context["command"] = imed_utils.get_command(args)
+    context["log_directory"] = imed_utils.get_log_directory(args)
```

Comment:
I think you don't actually need the `get_log_directory` function or `get_bids_path` function, the parser takes care of checking whether or not the argument is present via the `required` flag; this could just be:
```
context["log_directory"] = args.log_directory
context["bids_path"] = args.get_bids_path
```